themes/bootstrap3/templates/search/home.phtml

```diff
@@ -14,8 +14,10 @@
 ?>
 
 <div class="searchHomeContent">
-  <?=$this->context($this)->renderInContext("search/searchbox.phtml", ['ignoreHiddenFilterMemory' => true])?>
-  <?=$this->inlineScript(\Zend\View\Helper\HeadScript::SCRIPT, '$("#searchForm_lookfor").focus();', 'SET'); ?>
+  <?php $this->slot('search-home')->start() ?>
```

Comment:
What is the purpose of this slot? Just wondering if it should be named more specifically, since it currently impacts the search box on the home page, but not the whole page (the content block output below renders the bulk of the home page content).

Reply:
The purpose was to make it easier to replace the content in the center of the search home. A more specific name like `search-home-hero` may be appropriate. Also, a wiki page detailing all the slots might be helpful.

Reply:
We should discuss this further; I think this might be working at cross purposes with the content block mechanism.

I agree that a wiki page is a good idea; I've just added a TODO checkbox about that.